api/src/main/java/org/openmrs/Location.java

```diff
@@ -268,6 +270,22 @@ public void setCountyDistrict(String countyDistrict) {
 		this.countyDistrict = countyDistrict;
 	}
 	
+	/**
+	 * @return Returns the type.
+	 */
+	@Override
```

Comment:
```suggestion
```

Reply:
Generally, It would be good to have some actual Javadocs here, e.g.

```java
/**
 * @return Returns the code indicating the type of location this is
 * @since 2.5.0
 */